# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../unionfs/compile.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- unionfs-1.0.13/Makefile.vanilla	2005-08-10 00:36:55.000000000 +0200
+++ unionfs-1.0.13/Makefile	2005-08-10 00:37:06.000000000 +0200
@@ -68,7 +68,7 @@
 	${CC} -o $@ $^ ${UCFLAGS}
 
 unionimap: unionimap.c usercommon.c
-	${CC} -o $@ -luuid $^ ${UCFLAGS}
+	${CC} -o $@ $^ ${UCFLAGS} -luuid
 
 memtest:
 	perl match-malloc.pl log.txt
